cns/service/main.go

```diff
@@ -82,6 +82,8 @@ import (
 	ctrlzap "sigs.k8s.io/controller-runtime/pkg/log/zap"
 	ctrlmgr "sigs.k8s.io/controller-runtime/pkg/manager"
 	ctrlmetrics "sigs.k8s.io/controller-runtime/pkg/metrics/server"
+	"github.com/Azure/azure-container-networking/cns/grpc"
+	pb "github.com/Azure/azure-container-networking/cns/grpc/proto" 
```

Comment:
nitpick: make sure your Go formatter is set up so that these import lines get organized/sorted automatically

Reply:
I have updated my settings to support the Go formatter and sorted the lines on all the file I made changes to

Reply:
use `gofumpt`, if you aren't already